src/hooks/worlds/useOwnWorlds.ts

```diff
@@ -0,0 +1,36 @@
+import { useFirestore, useFirestoreCollectionData } from "reactfire";
+import { collection, query, where } from "firebase/firestore";
+
+import { ALWAYS_EMPTY_ARRAY, COLLECTION_WORLDS } from "settings";
+
+import { World } from "api/world";
+
+import { CONVERTER_WORLD_WITH_ID } from "utils/converters";
+import { WithId } from "utils/id";
+
+import { useUser } from "hooks/useUser";
+
+export const useOwnWorlds = (): {
+  isLoading: boolean;
+  ownWorlds: WithId<World>[];
+} => {
+  const firestore = useFirestore();
+  const { userId } = useUser();
```

Comment:
If you make this hook take a userId then this hook becomes simpler and doesn't need unusual defaults (like userId="") put into the query below

Reply:
Done in https://github.com/sparkletown/sparkle/pull/2820/commits/af825f995a7267c0da1cdbc23ea8630055576519